scripts/galaxy_main.py

```diff
@@ -0,0 +1 @@
+galaxy-main
```

Comment:
Maybe `git mv scripts/galaxy-main scripts/galaxy_main.py` and symlink `galaxy-main` to `galaxy_main.py` instead?

Reply:
Done that in a rebase!